api/src/opentrons/protocols/duration/errors.py

```diff
@@ -0,0 +1,5 @@
+class DurationEstimatorException(Exception):
+    def __init__(self, message):
+        super(Exception, self).__init__(
```

Comment:
I think this can just be:

```suggestion
        super().__init__(
```